app/src/main/java/com/vero/tasky/agenda/presentation/components/TimeDialog.kt

```diff
@@ -0,0 +1,26 @@
+package com.vero.tasky.agenda.presentation.components
+
+import androidx.compose.runtime.Composable
+import com.vanpra.composematerialdialogs.MaterialDialog
+import com.vanpra.composematerialdialogs.MaterialDialogState
+import com.vanpra.composematerialdialogs.datetime.time.timepicker
+import java.time.LocalTime
+
+@Composable
+fun TimeDialog(
+    dialogState : MaterialDialogState,
+    onTimePicked : (time: LocalTime) -> Unit
```

Comment:
Remember to add a modifier

Reply:
MaterialDialog (from the library) doesn't have a "modifier" parameter, so I can't use it anywhere

Reply:
Ah interesting

No problem then